x/gov/client/utils/query.go

```diff
@@ -212,28 +222,35 @@ func QueryProposerByTxQuery(
 	// support configurable pagination.
 	infos, err := tx.SearchTxs(cliCtx, cdc, tags, defaultPage, defaultLimit)
 	if err != nil {
-		return nil, err
+		return
```

Comment:
I believe it's best to still have explicit returns here.

```suggestion
		return p, err
```

Reply:
good call. Changed.